BaseApi.js

```diff
@@ -0,0 +1,65 @@
+const baseUrl = 'https://aplus-dev.liqd.net/api'
+
+const endpoints = {
+  ideas: baseUrl + '/modules/$moduleId/ideas',
+  liveQuestions: baseUrl + '/modules/$moduleId/interactiveevents/livequestions/'
+}
+
+const makeGetRequest = (url) => {
+    return fetch(url)
+      .then((response) => response.json())
+      .then((unpackedData) => {
+        return unpackedData;
+      })
+      .catch((error) => {
+        console.error(error);
+      });
+}
+
+const makePostRequest = (url, data = {}) => {
+     return fetch(url, {
+       method: 'POST',
+       headers: {
+           'Accept': 'application/json',
+           'Content-Type': 'application/json'
+       },
+       body: JSON.stringify(data)
+       })
+       .then((response) => response.json())
+       .then((responseData) => {
```

Comment:
we could return that newly created data here.

Reply:
Yeah I didnt really know what to do here, because we are not really using it yet. So you mean `return responseDate` here instead?

Reply:
yes exactly